library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -105,6 +106,26 @@ object ConstructorIo {
         return dataManager.getAutocompleteResults(query, params.toTypedArray())
     }
 
+    fun search(text: String, vararg facets: Pair<String, List<String>>, page: Int? = null, perPage: Int? = null, groupId: Int? = null): Observable<ConstructorData<SearchResponse>> {
```

Comment:
can we rename to `getSearchResults` to match `getAutocompleteResults`?

Reply:
yes